polyfill/test/duration.mjs

```diff
@@ -941,10 +941,13 @@ describe('Duration', () => {
       equal(`${d.round({ smallestUnit: 'years', relativeTo })}`, 'P6Y');
       equal(`${d.negated().round({ smallestUnit: 'years', relativeTo })}`, '-P6Y');
     });
-    it('balances differently depending on relativeTo', () => {
+    it('balances up differently depending on relativeTo', () => {
       const fortyDays = Duration.from({ days: 40 });
       equal(`${fortyDays.round({ largestUnit: 'years', relativeTo: '2020-01-01' })}`, 'P1M9D');
       equal(`${fortyDays.round({ largestUnit: 'years', relativeTo: '2020-02-01' })}`, 'P1M11D');
+      const minusForty = Duration.from({ days: -40 });
+      equal(`${minusForty.round({ largestUnit: 'years', relativeTo: '2020-01-01' })}`, '-P1M9D');
+      equal(`${minusForty.round({ largestUnit: 'years', relativeTo: '2020-02-01' })}`, '-P1M9D');
```

Comment:
It might be more useful to use 2020-03-01 and 2020-04-01 as the starting point where the durations would be different, as opposed to 2020-01-01 and 2020-02-01 where both cases only have to traverse 31-day months .

Reply:
I wanted to check that the results were different for the same `relativeTo`, but this is also a good point. I've done both in both cases now.